docs/source/_static/custom.js

```diff
@@ -8,7 +8,7 @@ document.addEventListener("DOMContentLoaded", function () {
     script.setAttribute("version", "stable");
     script.setAttribute("runllm-keyboard-shortcut", "Mod+j"); // cmd-j or ctrl-j to open the widget.
     script.setAttribute("runllm-name", "vLLM");
-    script.setAttribute("runllm-position", "BOTTOM_RIGHT");
+    script.setAttribute("runllm-position", "BOTTOM_LEFT");
```

Comment:
can you keep this and set the version selector in the left?

Reply:
After reverting and looking into this, I took the conversation back to the issue: https://github.com/vllm-project/vllm/issues/10089#issuecomment-2465357984 because I think it's worth discussing a bit more